static/js/public/first-snap-flow.js

```diff
@@ -1,39 +1,38 @@
-/* globals ClipboardJS */
+/* globals ClipboardJS, ga */
 
-import 'whatwg-fetch';
+import "whatwg-fetch";
```

Comment:
Aaaa, prettier changes again...

Reply:
:+1: this is the only branch that should be affected, so once these changes are made, we should be good